cdci_data_analysis/flask_app/app.py

```diff
@@ -223,6 +223,19 @@ def remove_nested_keys(D, keys):
     return D
 
 
+def sanitize_dict_before_log(dict_to_sanitize):
+    sensitive_keys = ['token']  # Add any other sensitive keys here
+    allowed_characters = r'[^A-Za-z0-9 ]'
```

Comment:
This regex matches NOT allowed characters, right?

Reply:
yes,I will rename the regex var